cog/api/gift.py

```
@@ -0,0 +1,64 @@
+            "color": 3447003,  # （藍色）
+            "description": ":gift:",
+        }
+        button = {
+            "type": 1,
+            "components": [
+                {
+                    "type": 2,
+                    "label": "前往確認",
+                    "style": 5,  # `5` 表示 Link Button
+                    "url": "https://store.scaict.org",  # 要導向的連結
+                }
```

Ruff

Trailing comma missing

Add trailing comma

(COM812)